platform/admin/core/admin-core-api/pom.xml

```diff
@@ -66,6 +66,26 @@
             <artifactId>security-core-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ddf.lib</groupId>
+            <artifactId>common-system</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.utils</artifactId>
+            <version>1.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.security.certificate</groupId>
+            <artifactId>security-certificate-keystoreeditor</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ddf.security.certificate</groupId>
+            <artifactId>security-certificate-generator</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
These dependencies should be placed in your own module, they can't be in admin-core-api. This is going to create a circular dependency to security that should not exist.